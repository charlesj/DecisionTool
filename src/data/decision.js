import { Map } from 'immutable'

export const getInitialState = () => {
  return Map()
}
