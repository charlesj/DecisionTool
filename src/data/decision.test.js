import { getInitialState } from './decision'

it("gives the expected initial state", () => {
  expect(getInitialState()).toMatchSnapshot()
})
