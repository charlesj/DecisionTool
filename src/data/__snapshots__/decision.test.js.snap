// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gives the expected initial state 1`] = `Immutable.Map {}`;
